src/lib/registration/ui/migration_repos_workflow.rb

```diff
@@ -335,11 +335,22 @@ def load_migration_products_offline(activations)
       def load_migrations_for_products(products, remote_product)
         log.info "Loading offline migrations for target product: #{remote_product.inspect}"
         log.info "Installed products: #{products.inspect}"
+
         self.migrations = registration_ui.offline_migration_products(products, remote_product)
 
         if migrations.empty?
-          # TRANSLATORS: Error message
-          Yast::Report.Error(_("No migration product found."))
+          msg = [
+            # TRANSLATORS. Error message
+            _("No migration product found."),
+            # TRANSLATORS: Help message, %{product} is the product name
+            _("Please, boot the original system and make sure " \
+              "that all registerable products are correctly registered.\n" \
+              "Also check that the installed system is supported for upgrade to \n" \
+              "%{product}.") % { product: remote_product }
```

Comment:
@lslezak 

`remote_product` is an `OpenStruct`. The message looks _weird_.

![Screenshot_openSUSE-Leap_2019-06-13_09:06:06](https://user-images.githubusercontent.com/1691872/59414991-8f5a3d00-8dba-11e9-9b9a-93424d9ec888.png)


What do you think about change it to something more human readable?

Like

```ruby
{ product: "#{remote_product.identifier} #{remote_product.version}"
```

which will produce the `SLED 15.1` output.

Reply:
Oh, I overlooked that...

Then I'd suggest using the `display_name` attribute, it contains a human readable product name like "SUSE Linux Enterprise Server 15 SP1" which is better than the internal identifier.

```suggestion
              "%{product}.") % { product: Y2Packager::ProductUpgrade.new_base_product.display_name }
```

(Not tested, please test it to make sure it really works as expected. :wink: )

Reply:
Nice! It works.

I'll update the `Screenshot` section in the PR's description.

Thanks!